formula-android/src/main/java/com/instacart/formula/integration/internal/BackStackUtils.kt

```diff
@@ -1,37 +1,6 @@
 package com.instacart.formula.integration.internal
 
-import com.instacart.formula.integration.BackStack
-import com.instacart.formula.integration.LifecycleEvent
-
 internal object BackStackUtils {
-    /**
-     * Takes last and current active contract state,
-     * and calculates attach and detach effects
-     */
-    fun <Key> findLifecycleEffects(
```

Comment:
Not needed anymore after moving to `Formula` APIs for contract integration binding.